src/data_pipeline.py

```diff
@@ -550,6 +550,11 @@ def main():
     hourly_consumed_calc.run()
     hourly_consumed_calc.output_results()
 
+    # move the log file into the specific year output folder
+    shutil.move(
+        outputs_folder("data_pipeline.log"), outputs_folder(f"{year}/data_pipeline.log")
+    )
```

Comment:
This will work, but it's probably easier to move the `configure_root_logger` call inside of `main` as you suggested.